devenv/config.toml

```diff
@@ -0,0 +1,6 @@
+[python]
+version = '3.8.18'
+darwin_x86_64 = 'https://github.com/indygreg/python-build-standalone/releases/download/20231002/cpython-3.8.18+20231002-x86_64-apple-darwin-install_only.tar.gz'
+darwin_x86_64_sha256 = 'fcf04532e644644213977242cd724fe5e84c0a5ac92ae038e07f1b01b474fca3'
+darwin_arm64 = 'https://github.com/indygreg/python-build-standalone/releases/download/20231002/cpython-3.8.18+20231002-aarch64-apple-darwin-install_only.tar.gz'
+darwin_arm64_sha256 = '1825b1f7220bc93ff143f2e70b5c6a79c6469e0eeb40824e07a7277f59aabfda'
```

Comment:
the linux ones should be in here too

and yeah I don't really like toml -- ini is simpler and doesn't require a dependency to read

Reply:
ah i forgot i initially chose toml cause i was going for 

```
python {
   version1
     ...
   version2
}
```

ini is the obvious choice now